src/utils/fetchLogFile/index.ts

```diff
@@ -1,7 +1,11 @@
+type IncompleteDownloadReason = "SERVER_ERROR" | "FILE_TOO_LARGE";
```

Comment:
Any reason not to use a string enum here?

Reply:
No, it was a preference but I updated it to use an enum.